ProjectSidewalk/SidewalkWebpage
# fixes some security vulnerabilities

Resolves #1781 

We have a few javascript libraries that have security vulnerabilities that we need to update. These include shelljs, diff, js-yaml, and marked. I had the packages updated automatically using `npm audit fix`. I then tested that the website still works given these changes.

Testing instructions:
1. Make sure your Docker container is not currently running by running `docker ps -a`. If there is an image called `projectsidewalk/web`, run `docker rm <container_id>`.
1. Delete the Docker image for the web container. Run `docker images` to get the image ID for `projectsidewalk/web`, then run `docker rmi <image_id> -f`.
1. Check out the 1781-fix-security-vulnerabilities branch.
1. Run `make dev` and then `npm start` to set up the new container.
1. Test that the website works by visiting the audit page, validation page, etc.

Note that you don't have to reload the database and such b/c we only have to remove the image for the website, the database container can remain intact.
